dataline-workers/src/main/java/io/dataline/workers/process/DockerProcessBuilderFactory.java

```diff
@@ -53,6 +53,7 @@ public ProcessBuilder create(final Path jobRoot, final String imageName, final S
         Lists.newArrayList(
             "docker",
             "run",
+            "-i",
```

Comment:
How did any of the tests work without this to accept inputs?

Reply:
Talked with Sherif about this, the end to end sync worker tests used it before the refactor:
https://github.com/datalineio/dataline/pull/113/files#diff-2f74d0920bf4620ecc493cb4c40c5ac9L106

And it looks like `-i` was used for both taps and targets even though it was only really necessary for targets.